src/jit/ByteCodeParser.cpp

```diff
@@ -1958,6 +1961,51 @@ static void compileFunction(JITCompiler* compiler)
             operands[3] = STACK_OFFSET(atomicRmwCmpxchg->dstOffset());
             break;
         }
+        case ByteCode::AtomicFenceOpcode: {
+            Instruction* instr = compiler->append(byteCode, Instruction::AtomicFence, opcode, 0, 0);
```

Comment:
Is there no other similar opcode? Data drop maybe?

Reply:
I put it together with `UnreachableOpcode`, which is also a 0-0 argument non-callback instruction (Data Drop is marked as a callback)